api/v1/installation_types.go

```diff
@@ -1048,3 +1054,68 @@ type WindowsNodeSpec struct {
 	// +optional
 	VXLANAdapter string `json:"vxlanAdapter,omitempty"`
 }
+
+type Proxy struct {
+	// HTTPProxy defines the value of the HTTP_PROXY environment variable that will be set on Tigera containers that connect to
+	// destinations outside the cluster.
+	// +optional
+	HTTPProxy string `json:"httpProxy,omitempty"`
+
+	// HTTPSProxy defines the value of the HTTPS_PROXY environment variable that will be set on Tigera containers that connect to
+	// destinations outside the cluster.
+	// +optional
+	HTTPSProxy string `json:"httpsProxy,omitempty"`
```

Comment:
Not to throw a wrench into things, but since these vars can take a separated list of items should we make them lists in our API?

```
httpsProxy:
- foo.com
- bar.org
```

and combine them? Or nah?

Reply:
`httpProxy` and `httpsProxy` should always be a single value, not a list AFAIK. `noProxy` can be a list of exempted destinations, but it's expected that this is a comma-separated list. If we ask for an array, it will likely be an inconvenience for the user IMO - they've likely been passing a comma-separated list elsewhere

Reply:
ah right, it was noProxy I was thinking about. I'm fine with the stance that it should just be the same as the env var.